### PR TITLE
Floodfill

### LINKED ISSUE #1233: Flood fill algorithm
This algorithm uses graphics library to fill the polygon with color.

### PR DESCRIPTION
**Pull request by robotjellyzone**
Fixes #1233

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- No changes 
- New algo added

#### Languages Used:
- C
- C++

#### Files Added:
- floodFill.c
- floodFill.cpp

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: floodFill/floodFill.C [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	    // recursive call for right pixel fill
+	    floodFill (x, y + 1, old, newcol);
+
+	    // recursive call for left pixel fill
+	    floodFill (x, y - 1, old, newcol);
+
+	    // recursive call for bottom-right corner pixel fill
+        floodFill (x + 1, y + 1, old, newcol);
+
+	    // recursive call for top-right corner pixel fill
```
Comment (MastersAbh) [inline floodFill/floodFill.C]: Fix indentation

File: floodFill/floodFill.C [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    }
+}
+
+void main ()
+{
+    int gd = DETECT, gm;
+    int left, top, right, bottom;
+
+    printf ("Enter the left point: ");
+    scanf ("%d", & left);
```
Comment (MastersAbh) [inline floodFill/floodFill.C]: no space after &

File: floodFill/floodFill.cpp [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    // check current pixel is old_color or not
+    if (current == old)
+    {
+        delay (5);
+
+	    // put new pixel with new color
+	    putpixel (x, y, newcol);
+
+	    // recursive call for bottom pixel fill
```
Comment (MastersAbh) [inline floodFill/floodFill.cpp]: please fix indentation of the files.

**Comment by jainaman224** [comment]
Change file and folder name to `Flood_Fill`.

File: Flood_Fill/Flood_Fill.c [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    }
+}
+
+void main ()
+{
+    int gd = DETECT, gm;
+    int left, top, right, bottom;
+
+    printf ("Enter the left point: ");
+    scanf ("%d", &left);
```
Comment (MastersAbh) [inline Flood_Fill/Flood_Fill.c]: remove the space after printf and scanf

File: Flood_Fill/Flood_Fill.c [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+    printf ("Enter the top point: ");
+    scanf ("%d", &top);
+
```
Comment (MastersAbh) [inline Flood_Fill/Flood_Fill.c]: remove the space after printf and scanf

File: Flood_Fill/Flood_Fill.c [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    printf ("Enter the right point: ");
```
Comment (MastersAbh) [inline Flood_Fill/Flood_Fill.c]: remove the space after printf and scanf

**Comment by robotjellyzone** [comment]
Hi, @jainaman224 please review my pr as i am done with all the changes that were proposed by @MastersAbh

**Comment by robotjellyzone** [comment]
Hi @rishabhgarg sorry I can not as by mistake I have done some changes through github only instead of git command and so @jainaman224 said that he will squash it once get approval for my pr from mentors

**Comment by robotjellyzone** [comment]
> Hi @robotjellyzone , Please rename your file to Flood_Fill.c (c should be small).
> And why are 4 files in your PR ?
> I think both files got repeated with different names. Please remove the older files.

@MastersAbh  see this https://github.com/jainaman224/Algo_Ds_Notes/pull/1349#issuecomment-497203764

**Comment by marwahmanbir** [comment]
@robotjellyzone please remove the redundant files from your PR.

**Pull request merged**